db/re/item_randomopt_group.yml

```diff
@@ -12285,3 +12285,341 @@ Body:
             MinValue: 1
             MaxValue: 5
             Chance: 769
+  - Id: 145
```

Comment:
ID Conflict with https://github.com/rathena/rathena/pull/7207 Causing the [Error]: Unknown random option group "EIN_DDBOX".

Reply:
This conflict will be resolved later when either pull is merged to master